homeassistant/helpers/config_entry_flow.py

```diff
@@ -258,3 +262,149 @@ async def webhook_async_remove_entry(
         return
 
     await hass.components.cloud.async_delete_cloudhook(entry.data["webhook_id"])
+
+
+class HelperCommonFlowHandler:
```

Comment:
Let's add this to a new helper file. This one is getting overloaded.

Reply:
sounds good. any suggestion for a name? `basic_config_entry_flow.py`; `helper_config_entry_flow.py`; `schema_config_entry_flow.py`?

Reply:
Moved to `helper_config_entry_flow.py`